ynput/OpenPype
# Nuke: Render Workfile Attributes

## Changelog Description
`Workfile Dependency` default value can now be controlled from project settings.
`Use Published Workfile` makes using published workfiles for rendering optional.

## Testing notes:
1. Toggle `deadline/publish/NukeSubmitDeadline/workfile_dependency` off.
2. Create Write instance in Nuke and check the `Workfile Dependency` is off.
3. Toggle `Use Published Workfile` and validate the Deadline job is using the work area workfile.
